main_loop.h

```diff
@@ -187,14 +187,16 @@ class MainLoop {
       send_data.motor_encoder = status_.fast_loop.motor_position.raw;
       send_data.motor_position = status_.motor_position;
       send_data.joint_position = status_.output_position;
-      send_data.torque = status_.torque;
+      send_data.torque = status_.torque + t0;
```

Comment:
I would remove this function from main loop and do the logic in system instead

Reply:
This is because I want to receive the data with the zero offset